packages/tools/src/tools/base/AnnotationTool.ts

```diff
@@ -402,8 +402,27 @@ abstract class AnnotationTool extends BaseTool {
     property: string,
     annotation?: Annotation
   ): unknown {
+    // Since we would like to have a toolGroup settings and cornerstone core Settings
```

Comment:
in the api contract:

```
settings: Settings,
property: string,
annotation?: Annotation
```
It feels a bit weird to me that we pass in a settings that we may then not use. I think it may be easier for the consumer above to understand the following:

```
property: string,
annotation?: Annotation
```

Then, if annotation is present, we can get the annotation scoped settings via: 


```
const settings = Settings.getObjectSettings(
    annotation,
    this.getToolName()
);
```


And the toolGroupSettings can still be got via `this.toolGroupId`.